### ISSUE TITLE
New Container App routes not being served

### ISSUE THREAD

**Issue by Eein** [lotus/lotus#285]
I'm following the guides, and i'm having trouble creating a new container app with
`lotus generate app admin`
When I create the new app, everything is good. When I visit the newly generated route `/admin` it shows the bootstrapped screen.

If I make a new action under this app using
`lotus generate action app home#index`
all potential routes I could think of would not work

`
::1 - - [02/Jul/2015:00:40:09 -0500] "GET /admin/home HTTP/1.1" 404 - 0.2267
::1 - - [02/Jul/2015:00:40:14 -0500] "GET /admin/home/ HTTP/1.1" 404 - 0.2213
::1 - - [02/Jul/2015:00:40:28 -0500] "GET /home/ HTTP/1.1" 404 - 0.2177
::1 - - [02/Jul/2015:00:40:40 -0500] "GET /home HTTP/1.1" 404 - 0.2230`

I've reproduced this several times with a fresh install and following the instructions above.

EDIT:
potentially related to #262 

my config/environment.rb that is autogenerated by lotus
```ruby
Lotus::Container.configure do
  mount Admin::Application, at: '/admin'
  mount Web::Application, at: '/'
end
```

Lotus 0.4.0

ruby 2.2.1p85 (2015-02-26 revision 49769) [x86_64-darwin14]

OSX Yosemite 10.10.2

**Comment by mvader** [comment]
I followed the steps you stated and this is what I got:
```
::1 - - [02/Jul/2015:09:19:22 +0200] "GET / HTTP/1.1" 500 - 0.1697
::1 - - [02/Jul/2015:09:19:22 +0200] "GET /favicon.ico HTTP/1.1" 500 - 0.1102
::1 - - [02/Jul/2015:09:20:12 +0200] "GET / HTTP/1.1" 500 - 0.2046
::1 - - [02/Jul/2015:09:20:27 +0200] "GET / HTTP/1.1" 404 - 0.1687
::1 - - [02/Jul/2015:09:20:33 +0200] "GET /home HTTP/1.1" 200 - 0.1429
::1 - - [02/Jul/2015:09:20:38 +0200] "GET /admin/home HTTP/1.1" 200 - 0.1370
```

Lotus found my route and sent a 200 status.
Running lotus 0.4.0 and ruby 2.2.2p95 (2015-04-13 revision 50295) [x86_64-darwin14] on OS X Yosemite 10.10.4.

**Comment by joneslee85** [comment]
@Eein we also found a regression with version 0.4.0, this issue will be addressed in the next 0.5.0. For the time being, you could live on-edge with:

```
# Gemfile
gem 'lotusrb', github: 'lotus/lotus'
gem 'lotus-model', github: 'lotus/model'
gem 'lotus-controller', github: 'lotus/controller'
gem 'lotus-view', github: 'lotus/view'
gem 'lotus-helpers', github: 'lotus/helpers'
gem 'lotus-validations', github: 'lotus/validations'
gem 'lotus-router', github: 'lotus/router'
```

**Comment by jodosha** [comment]
@Eein Not in 0.5.0, but in a upcoming v0.4.1. /cc @joneslee85 @lotus/core-team
